packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/models/errors/error_model.dart

```diff
@@ -1,12 +1,12 @@
 {{> licence.dart }}
 
 import 'package:widget_toolkit/models.dart'
-show ErrorModel, L10nErrorKeyProvider;
+    show ErrorModel, L10nErrorKeyProvider;
 
-import '../../../assets.dart' show I18nErrorKeys;
+import '../../../assets.dart' show I18nErrorKeys, I18nFeatureAuthMatrixKeys;
```

Comment:
I think `I18nFeatureAuthMatrixKeys` should only be included if auth matrix is enabled.